peregrine/acquisition.py

```diff
@@ -471,7 +483,8 @@ def progress_callback(freq_num, num_freqs):
                                      carr_freq - self.IF,
                                      code_phase,
                                      snr,
-                                     status)
+                                     status,
+                                     'l1ca')
```

Comment:
The signal (l1ca, l2c, etc) of the AcquisitionResult should probably be passed into the `Acquisition` class, and driven by what is in `initSettings`. Can you update `Acquisition` and these functions so that the signal is passed through rather than hardcoded?